Bio/SeqIO/__init__.py

```diff
@@ -1049,6 +1049,16 @@ def convert(in_file, in_format, out_file, out_format, molecule_type=None):
     >EAS54_6_R1_2_1_443_348
     GTTGCTTCTGGCGTGGGTGGGGGGG
     <BLANKLINE>
+
+    Note some formats require a binary handle (bytes) rather than a text mode
+    handle (strings), and may require you to specify the molecule type when it
```

Comment:
This sounds a bit strange. Text/binary handles and molecule types are two independent things.

Reply:
Point taken, and since the examples here are using filenames anyway I had changed this bit to focus on the molecule type only.